docs/reatom-react.md

```diff
@@ -29,7 +29,7 @@ Connects the atom to the store represented in context and returns the state of t
 const atomValue = useAtom(atom)
 ```
 
-#### Depended value by selector
+#### Dependent value by selector
```

Comment:
Not sure what the original heading meant, please confirm

Reply:
@belozer "Dynamic atom, which will be re-created on dependency change" ?

Reply:
@MunGell hard to understand

Reply:
Maybe with description like that:

#### Dynamic atom

Dynamic atoms are re-created on dependency change (passed as the second argument). This technique is useful when "<something>".

<code>

Reply:
Maybe like: **With selector (render on value change)**

Reply:
I've updated the wording there, please check